### ISSUE TITLE
Cannot require `clojure.tools.reader`

### ISSUE THREAD

**Issue by moxaj** [anmonteiro/lumo#209]
With latest lumo (1.6):

```clojure
cljs.user=> (require 'clojure.tools.reader)
No such macros namespace: cljs.tools.reader, could not locate cljs/tools/reader.clj or cljs/tools/reader.cljc
```

However, the following works:

```clojure
cljs.user=> (require 'cljs.tools.reader)
nil
```

**Comment by moxaj** [comment]
I suspect https://dev.clojure.org/jira/browse/CLJS-2069 may be relevant.

**Comment by moxaj** [comment]
Upstream bug: https://dev.clojure.org/jira/browse/CLJS-2266

**Comment by anmonteiro** [comment]
Still an issue in Lumo even if also upstream.

**Comment by moxaj** [comment]
Oh, sorry, not familiar with the policies :)

**Comment by anmonteiro** [comment]
fixed in 1.7.0
